hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsVolumeList.java

```diff
@@ -62,9 +63,13 @@
   private final VolumeChoosingPolicy<FsVolumeImpl> blockChooser;
   private final BlockScanner blockScanner;
 
+  private boolean enableSameDiskTiering;
+  private MountVolumeMap mountVolumeMap;
```

Comment:
These two fields can be declared as "final"

Reply:
+1, will fix